### DISCUSSION TITLE
NodeDndPanel 节点拖拽面板 可以同时存在两个吗？想实现一个tabs的功能

### DISCUSSION THREAD

**Discussion by cyh-88** [antvis/XFlow#358] [Q&A]
![image](https://user-images.githubusercontent.com/55520060/222684941-1b67d96b-0593-49ae-b27c-86ed7421af7e.png)

创建了两个NodeDndPanel，一个有数据 另一个没数据，看获取数据的请求只走了一次，这是为什么呐

**Comment by lyn-boyu** [comment]
![image](https://user-images.githubusercontent.com/47809781/223313984-babc1ec0-91f3-42bc-9d05-50c7f575c486.png)

不支持多实例哈

**Reply by cyh-88** [reply]
![image](https://user-images.githubusercontent.com/55520060/223345234-77ab860c-a78f-43a5-902d-0e7a8e83d4d1.png)

在外部可以通过什么方法可以去改变内部的数据呐。我把这个实力销毁再进行渲染service也不会从去新执行

**Reply by zhangjiana** [reply]
你好，请问这个问题解决了吗？最后怎么实现的

**Comment by SoarBlueSky** [comment]
你可以自己手写啊，然后调用xflow的api

**Reply by zhangjiana** [reply]
怎么手写呢？大佬指教一下
